test/e2e/nullable.ts

```diff
@@ -0,0 +1,74 @@
+export const input = {
+  type: 'object',
+  properties: {
+    a: {
+      const: '',
+      nullable: true,
+    },
+    b: {
+      const: null,
+      nullable: true,
+    },
+    c: {
+      enum: ['a', 'b'],
+      nullable: true,
+    },
+    d: {
+      enum: ['', null],
+      nullable: true,
+    },
+    e: {
+      type: 'string',
+      nullable: true,
+    },
+    f: {
+      type: 'null',
+      nullable: true,
+    },
+    g: {
+      type: 'string',
+      const: '',
+      nullable: true,
+    },
+    h: {
+      type: 'string',
+      const: null,
+      nullable: true,
+    },
```

Comment:
@mdmower-csnw: This might not be directly related to the changes in this PR, but I was wondering if there's a warning for a schema like this where the `type` and `const` (or `enum` values) are inconsistent. I don't know how AJV handles such a schema.